### DISCUSSION TITLE
Filter by issue #

### DISCUSSION THREAD

**Discussion by liserubarth** [community/community#10380] [Projects and Issues]
We often need to filter a list or view to find an issue by the issue number. Apparently this feature has been removed in the current beta. This is a very important request from all of our teams' perspective, etc. With hundreds of issues over the board we rely on issue numbers. The old version supported this feature. Thanks.

**Comment by k-doering-NOAA** [comment]
It would be great to have issue number available as one of the columns in project beta.
